samples/meetings-transcription/csharp/MeetingTranscription/Controllers/HomeController.cs

```diff
@@ -0,0 +1,65 @@
+﻿// <copyright file="HomeController.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+using MeetingTranscription.Helpers;
+using MeetingTranscription.Models.Configuration;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using System.Collections.Concurrent;
+using System.Threading.Tasks;
+
+namespace MeetingTranscription.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly ConcurrentDictionary<string, string> transcriptsDictionary;
+
+        /// <summary>
+        /// Stores the Azure configuration values.
+        /// </summary>
+        private readonly IOptions<AzureSettings> azureSettings;
+
+        /// <summary>
+        /// Stores the Azure configuration values.
+        /// </summary>
+        private readonly GraphHelper graphHelper;
+
+        public HomeController (ConcurrentDictionary<string, string> transcriptsDictionary, IOptions<AzureSettings> azureSettings)
+        {
+            this.transcriptsDictionary = transcriptsDictionary;
+            this.azureSettings = azureSettings;
+            graphHelper = new(azureSettings);
+        }
+
+        /// <summary>
+        /// Returns view to be displayed in Task Module.
+        /// </summary>
+        /// <param name="meetingId">Id of the meeting.</param>
+        /// <returns></returns>
+        public async Task<IActionResult> Index([FromQuery] string meetingId)
+        {
+            ViewBag.Transcripts = "Transcript not found.";
+
+            if (meetingId != null)
```

Comment:
Might want to check for null or empty. string.isNullOrEmpty(meetingId).

Reply:
Added